api/handler/organization.go

```diff
@@ -87,19 +87,27 @@ func (h *OrganizationHandler) Index(ctx *gin.Context) {
 // @Accept       json
 // @Produce      json
 // @Param        name path string true "name"
+// @Param        body body types.DeleteOrgBody true "body"
 // @Success      200  {object}  types.Response{} "OK"
 // @Failure      500  {object}  types.APIInternalServerError "Internal server error"
 // @Router       /organizations/{name} [delete]
 func (h *OrganizationHandler) Delete(ctx *gin.Context) {
-	name := ctx.Param("name")
-	err := h.c.Delete(ctx, name)
+	var req types.DeleteOrgReq
+	if err := ctx.ShouldBindJSON(&req); err != nil {
+		slog.Error("Bad request format", "error", err)
+		ctx.JSON(http.StatusBadRequest, gin.H{"message": err.Error()})
```

Comment:
change to `httpbase.BadRequest`

Reply:
Thanks for your review. Changed to use `httpbase.BadRequest`.